.github/workflows/version.yml

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
rename the file to `api-version.yml` to avoid mixing with existing OT version.

Reply:
I tend to keep this name for extensibility. Thoughts?